pages/index.js

```diff
@@ -8,6 +8,7 @@ import runLegacyScripts from '@/components/legacy';
 import { useEffect }  from 'react';
 import $ from "jquery";
 import { useRouter } from 'next/router';
+import ContactUsForm from '@/components/ContactUS';
```

Comment:
Be mindful of character case.

```suggestion
import ContactUsForm from '@/components/ContactUs';
```